react-sample/src/Counter.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react"
+import PropTypes from "prop-types"
+
+function Counter({ initialCount  } : { initialCount: number}) {
+  const [count, setCount] = useState(initialCount)
+
+  function incrementCount() {
+    setCount(count + 1)
+    console.log(count)
+  }
+
+  function decrementCount() {
+    setCount(count - 1)
+    console.log(count)
+  }
+
+  return (
+    <div>
+      <button onClick={decrementCount}>-</button>
+      {count}
+      <button onClick={incrementCount}>+</button>
+    </div>
+  )
+}
+
+Counter.propTypes = {
+  initialCount: PropTypes.number
+}
+
+export default Counter
\ No newline at end of file
```

Comment:
**GPT COMMENT**: 
 - It's a good practice to add a newline at the end of the file to avoid any possible compilation issues and to adhere to the POSIX standards.